config/optional-features.json

```
@@ -0,0 +1,5 @@
+{
+  "application-template-wrapper": false,
+  "jquery-integration": false,
```

```suggestion
  "jquery-integration": true,
```
`e-c-mirage` still relies in jquery AFAIK